gnuradio.lwr

```diff
@@ -37,7 +37,13 @@ depends:
 - python-zmq
 - python-click-plugins
 - python-pyqtgraph
+- pyyaml
+- pygobject
+- gobject-introspection
+- gtk3
 - mako
+- doxygen
+- orc
```

Comment:
ORC is not a dependency of GNU Radio -- it's an optional dep of VOLK, IIRC.

Reply:
I agree that ORC is an optional VOLK dependency but as there is no separate VOLK recipe and VOLK is installed as gnuradio's git submodule I've decided to add it in the gnuradio recipe. 

I just removed 7de3e850c3f12f7c2bfc30b8f659e23e199ca530 from this PR not not block the other changes. Let me know if you want ORC to not be installed by pybombs at all or you prefer a separate PR for it.